Add notes for chapter 01

lucio-cornejo opened the pull request.

Commit:
ignore js file for live testing examples

Commit:
reduce default font size

Commit:
add instructions for presenting

Commit:
add notes for introduction chapter

jonthegeek commented on .gitignore:
```diff
@@ -10,3 +10,5 @@ bookclub-js4ds.knit.md
 libs
 *.html
+
+live-examples.js
```
```suggestion
live-examples.js

```

jonthegeek commented on index.Rmd:
```diff
@@ -35,3 +35,23 @@ This companion follows the [R4DS Online Learning Community Code of Conduct](http
+    1. From your forked repo, **create a branch** for your notes of the chapter to present.
+    1. After saving your finished chapter notes, **add**, **commit** and **push**
+    your changes into your forked repo.
+    1. Finally, create a **pull request** in order to update the online version of the book.
```
```suggestion
    1. Finally, create a **pull request** in order to update the online version of the book.
    1. Pay attention to the messages on GitHub. For example, if your branch is out of date, please merge the main branch and resolve any conflicts (see the [full instructions](https://github.com/r4ds/bookclub-js4ds#how-to-present) for easy ways to deal with things in R).
```

jonthegeek reviewed (dismissed):
Looks good!

I'm making a couple small changes, but I'll commit them myself.

jonthegeek committed:
Add newline back to .gitignore.

jonthegeek committed:
Encourage good GitHub behavior.

The reason for the full instructions is that people create merge conflicts that make things harder to deal with. usethis helps to avoid such conflicts.

jonthegeek reviewed (approved):
Looks good as long as I didn't break the build!
